src/components/screens/ExperimentEditor.vue

```diff
@@ -541,7 +752,7 @@ export default {
       }
 
       const editedItem = _.merge({}, this.item || {}, editedFields)
-
+      console.log(editedItem)
```

Comment:
Can remove

Reply:
Agree. I think I added it when I debug something and forgot it.